packages/snap/src/rpc.ts

```diff
@@ -29,14 +29,36 @@ export async function getPublicKey(request: AntelopeRequest): Promise<string> {
   return String(await derivePublicKey(chain));
 }
 
+const MAX_TRANSACTION_LENGTH = 10000;
```

Comment:
I'm wondering if anyone has thoughts on whether or not this is a good max length 🤔